src/logic/calculate.js

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable quotes */
```

Comment:
### Optional
- [ ] Here also, as you can see from the required code we were to copy, you could see single quotes instead of double we have currently

Reply:
Hi @addod19 

Thank you for your reviewing my work, much appreciated.
I disabled the linter on the quotes because my linters were enforcing a single quote rule on the calculate and operate.js files. So to avoid editing the files I decided to disable the single quote rule just for these two files.But i will make sure to remove the remove this on my next PR.